SimTracker/SiPhase2Digitizer/plugins/Phase2TrackerDigitizerAlgorithm.cc

```diff
@@ -37,25 +37,14 @@
 using namespace edm;
 using namespace sipixelobjects;
 
-namespace {
+namespace Ph2TkDigiAlgo {
```

Comment:
"For namespaces use lower case" (see 2.7 in http://cms-sw.github.io/cms_coding_rules.html)